_contentTemplates/common/calendar-header-template.md

```diff
@@ -0,0 +1,7 @@
+#header-template
+The `<HeaderTemplate>` allows you to control the header segment of the calendar. If the application defines this template none of the built-in buttons will rendered. 
+
+You can access the following fields directly through the `context`:
+* `Date` - the `DateTime` object that is used for data-binding
+* `View` - the current view of the calendar
```

Comment:
This should be updated as it is no longer valid:
https://github.com/telerik/blazor/issues/4572#issuecomment-1153556353